source/includes/_feature-extending.md

```diff
@@ -0,0 +1,39 @@
+## Merging and Joining Datasets
+
+```http
+POST /datasets/{dataset_id}/variables/ HTTP/1.1
+Content-Type: application/json
+
+{
+    "function": "adapt",
+    "args": [
+        {
+            "dataset": "/datasets/{other_id}/",
+            "filter": {},
+            "select": {
+                "function": "identify",
+                "args": [
+                    {
+                        "ids": [...]
+                    }
+                ]
+            }
+        },
+        {
+            "variable": "/datasets/{other_id}/variables/{right_key_id}/"
+        },
+        {
+            "variable": "/datasets/{dataset_id}/variables/{left_key_id}/"
+        },
+        ... // Other future args
+    ]
+}
+
+-----
+202 Continue
```

Comment:
The API is sync at the moment
https://github.com/Crunch-io/backend-http/blob/d77cc62182d165946180172be0d44c5e5b2156d5/cr/server/api/controllers/variables/catalog.py#L569